### PR TITLE
Add Teams Toolkit Support guide

### PR DESCRIPTION
**Pull request by zyxiaoyuer**
I added a row to the table for Teams Toolkit support

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/35b3a09a74264623c598702f499bddeaf4162928/msteams-platform/feedback.md) | 80 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/263067cc-2d11-42c1-8416-aa58b039952a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Microsoft Teams community of developers use Stack Overflow to connect with other
...
+|          **Teams Toolkit or TeamsFx SDK and CLI issues**           |     <ul><li> [Github Issues](https://github.com/OfficeDev/TeamsFx/issues) Create new issue on the [TeamsFx (Microsoft Teams Framework) GitHub repository](https://github.com/OfficeDev/TeamsFx) to report issues or raise feature request. We love Github Issues and our extensive support is assured for customers who ask questions there. </li>                                                                                                                                                                       <li>  [Stack Overflow](https://stackoverflow.com/questions/tagged/teams-toolkit): Use the `teams-toolkit` tag to post questions. You must follow the Stack Overflow guidelines, such as provide a descriptive title, a complete and concise problem statement, along with sufficient detail to reproduce your issue. Feature requests or broad questions are off-topic. If you are a new user, for more information, see Stack Overflow Help Center.  </li> </ul>                                                                                            |
```
Comment (v-abirade) [inline msteams-platform/feedback.md:18]: Please rephrase:
We love Github Issues and our extensive support is assured for customers who ask questions there.

**Review by v-abirade — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df13ea94369206a6356895175105d40b486d9606/msteams-platform/feedback.md) | 80 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f8ebbd58-b741-4018-9fe5-7790f12a2766) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/feedback.md (hunk 1 above)
Comment (zyxiaoyuer) [reply msteams-platform/feedback.md]: done rephrasing, please check.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed965eb1da1caafcb8b14514bff82b6a2c8e6c51/msteams-platform/feedback.md) | 79 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d616f646-d141-461d-9ff9-a105fd541011) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**